spec/features/publishable_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "spec_helper"
```

Comment:
Still thinking the "click_button and then redirect" should belong to a feature test.